src/braket/default_simulator/simulator.py

```diff
@@ -386,6 +388,39 @@ def _observable_hash(observable: Observable) -> Union[str, dict[int, str]]:
         else:
             return str(observable.__class__.__name__)
 
+    def _map_circuit_to_contiguous_qubits(self, circuit: Circuit) -> Circuit:
+        """
+        Maps the qubits in operations and result types to contiguous qubits.
+
+        Args:
+            circuit (Circuit): The circuit containing the operations and result types.
+
+        Returns:
+            Circuit: The circuit with qubits in operations and result types mapped 
+            to contiguous qubits.
+        """
+        qubit_map = self._contiguous_qubit_mapping(circuit.qubit_set)
```

Comment:
`_contiguous_qubit_mapping` is a static method, and this is the only line that uses `self`. You could turn the entire `_map_circuit_to_contiguous_qubits` into a class method with the signature 
```python
@classmethod
def _map_circuit_to_contiguous_qubits(cls, circuit: Circuit) -> Circuit:
...
qubit_map = cls._contiguous_qubit_mapping(circuit.qubit_set)
```